docs/book/src/reference/reference.md

```diff
@@ -27,6 +27,8 @@
       - [Object/DeepCopy](markers/object.md)
       - [RBAC](markers/rbac.md)
 
+  - [Performance Profiling with Pprof](../cronjob-tutorial/pprof-tutorial.md)
```

Comment:
It should be under reference and not (../cronjob-tutorial/pprof-tutorial.md)

Reply:
Updated!